### DISCUSSION TITLE
Ask me Anything with Marc

### DISCUSSION THREAD

**Discussion by clemra** [langfuse/langfuse#2632] [Announcements]
**Ask me Anything with Marc**

EDIT: The recording is now [live on youtube](https://www.youtube.com/watch?v=gC9bXko2vFM)

Hi all! 

After our [Town Hall](https://www.youtube.com/watch?v=WGERHcRnBYQ), we're hosting our next public session. Come and say hi to @marcklingen, CEO and co-founder of Langfuse. Marc is doing an AMA (ask me anything) session + will update on what's going on in Langfuse-Land. 

When: Wednesday, 24th of July, noon PT, 9pm CET
Where: [Sign Up](https://lu.ma/8utmvnlv) - _takes place virtually_

Looking forward to seeing you there!

**Comment by marcklingen** [comment]
Really looking forward to this, last time was fun!

**Comment by clemra** [comment]
Giving this a push - we're looking forward to seeing you all on Wednesday! Over 40 sign-ups already! 
Sign Up: https://lu.ma/8utmvnlv

**Comment by clemra** [comment]
The recording is now [live on youtube](https://www.youtube.com/watch?v=gC9bXko2vFM)
